mbf_costmap_nav/src/nav_core_wrapper/wrapper_local_planner.cpp

```diff
@@ -71,7 +71,7 @@ bool WrapperLocalPlanner::setPlan(const std::vector<geometry_msgs::PoseStamped>
 
 bool WrapperLocalPlanner::cancel()
 {
-  return false;
```

Comment:
Oops forgot to remove. Initially I had the publishZeroVelocity command in the cancel() function of the abstractControllerExectuion class, and needed this to return true so that it could finish that function. But the issue was that that cancel function would publish zero before the thread stopped, making another velocity publish after the zero.